src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -160,7 +160,7 @@ private Card createJobStartedCard() {
 
         addCauses(factsList);
 
-        String activityTitle = "Update from build " + jobName + ".";
+        String activityTitle = "Update from " + jobName + ".";
```

Comment:
I still think the title could be better for started, but feedback would be appreciated